pkg/aws/aws_client/env.go

```diff
@@ -0,0 +1,16 @@
+package aws_client
+
+import "os"
+
+func ENVCredential() bool {
+	if os.Getenv("AWS_ACCESS_KEY_ID") != "" && os.Getenv("AWS_SECRET_ACCESS_KEY") != "" {
+		return true
+	}
+	return false
+}
+func ENVAWSProfile() bool {
```

Comment:
Do these need to be exported outside their package?

Also a NP, in CS we camelcase naming even for abbreviations to help with readability, might be worth following along with that here, `ENVAWSProfile` -> `envAwsProfile`

Reply:
Updated the name